### PR TITLE
OSDOCS-1781: Document image prune ignoreInvalidImageReferences parameter

### PR DESCRIPTION
**Pull request by bmcelvee**
https://issues.redhat.com/browse/OSDOCS-1781

Preview build: https://deploy-preview-37229--osdocs.netlify.app/openshift-enterprise/latest/applications/pruning-objects?utm_source=github&utm_campaign=bot_dp#pruning-images_pruning-objects

### CODE REVIEW

**Comment by bmcelvee** [comment]
@dmage would you mind reviewing this PR, please? Thank you!

I made some style updates to match our current documentation guidelines. But here are the lines with new content: 
[Sample YAML entry] https://github.com/openshift/openshift-docs/pull/37229/files#diff-1371611b6222142b5fa8aeb2d65a0090e869b8fe6a74320aef259fb55da34b00R35

[Detailed callout] https://github.com/openshift/openshift-docs/pull/37229/files#diff-1371611b6222142b5fa8aeb2d65a0090e869b8fe6a74320aef259fb55da34b00R66

This line is commented out, but I wanted to cover all the the bases. I think the `ignoreInvalidImageReferences` parameter applies only to the automated and not manual pruner, is that correct? 
[Manual pruning updated that is currently commented out] https://github.com/openshift/openshift-docs/pull/37229/files#diff-d41b3ad2d3e956ff2fb324664a58c462c4d4473a49cc1fa82920321897fedebfR204

**Comment by wzheng1** [comment]
@xiuwang Could you please help to review? Thanks!

File: modules/pruning-images.adoc [hunk 1]
```diff
@@ -50,23 +52,59 @@ status:
...
-<11> `failedJobsHistoryLimit`: The maximum number of failed jobs to retain. Must be `>= 1` to ensure metrics are reported. This is an optional field, default is `3`. The initial value is `3`.
-<12> `observedGeneration`: The generation observed by the Operator.
-<13> `conditions`: The standard condition objects with the following types:
-* `Available`: Indicates if the pruning job has been created. Reasons can be Ready or Error.
-* `Scheduled`: Indicates if the next pruning job has been scheduled. Reasons can be Scheduled, Suspended, or Error.
-* `Failed`: Indicates if the most recent pruning job failed.
+<1> Optional. `CronJob` formatted `schedule`. The default setting is daily at midnight.
+<2> Optional. If `suspend` is set to `true`, the `CronJob` running pruning is suspended. The `suspend` field is set to `false` by default. The initial value on new clusters is `false`.
+<3> Optional. `keepTagRevisions` is the number of revisions per tag to keep. The  default value is `3`. The initial value is `3`.
```
Comment (xiuwang) [inline modules/pruning-images.adoc:57]: @bmcelvee We may need add logLevel here like other spec parameters
logLevel sets the level of log output for the pruner job. Valid values are: "Normal", "Debug", "Trace", "TraceAll". Defaults to "Normal".



logLevel sets the level of log output for the pruner job. Valid values are: "Normal", "Debug", "Trace", "TraceAll". Defaults to "Normal".
--
 

<br class="Apple-interchange-newline">

File: modules/pruning-images.adoc [hunk 2]
```diff
@@ -50,22 +52,24 @@ status:
...
+<9> Optional. `tolerations` is the standard pod tolerations.
+<10> Optional. `successfulJobsHistoryLimit` is the maximum number of successful jobs to retain. The value must be  set to `>= 1` to ensure metrics are reported. The default is `3`. The initial value is `3`.
+<11> Optional. `failedJobsHistoryLimit`: The maximum number of failed jobs to retain. Must be `>= 1` to ensure metrics are reported. This is an optional field, default is `3`. The initial value is `3`.
+<12> Optional. `ignoreInvalidImageReferences` indicates whether the pruner can ignore errors while parsing image references. The default value is `True`.
+<13> Optional. `observedGeneration` is the generation observed by the Operator.
+<14> Optional. `conditions`: The standard condition objects with the following types:
+
+- `Available`: Indicates if the pruning job has been created. Reasons can be `Ready` or `Error`.
+- `Scheduled`: Indicates if the next pruning job has been scheduled. Reasons can be `Scheduled`, `Suspended`, or `Error`.
+- `Failed`: Indicates if the most recent pruning job failed.
```
[the remark this answers is not in the document]
Comment (xiuwang) [reply modules/pruning-images.adoc:72]: @bmcelvee Yes, IMO, we can add Degraded here.

**Comment by bergerhoffer** [comment]
The `branch/enterprise-4.11` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.10. And any PR going into `main` must also target the latest version branch (enterprise-4.11).

If the update in your PR does NOT apply to version 4.11 onward, please retarget this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Pull request closed without merging**